internal/services/network/network_connection_monitor_resource_test.go

```diff
@@ -92,7 +92,8 @@ func testAccNetworkConnectionMonitor_addressUpdate(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.ImportStep(),
+		// todo investigate with framework
+		// data.ImportStep(),
```

Comment:
I think the change that was causing this issue has been reverted now so this can be uncommented

Reply:
I think this needs to stay in as I'm getting this error on main. I think there is something happening with Sets being weird

```
=== RUN   TestAccNetworkConnectionMonitor_addressUpdate
    testcase.go:137: Step 3/3 error running import: ImportStateVerify attributes not equivalent. Difference is shown below. The - symbol indicates attributes missing after import.

          map[string]string{
        + 	"endpoint.1.coverage_level":       "",
        + 	"endpoint.1.target_resource_id":   "",
        + 	"endpoint.1.target_resource_type": "",
          }
```